sycl/source/handler.cpp

```diff
@@ -1861,8 +1861,13 @@ std::shared_ptr<ext::oneapi::experimental::detail::graph_impl>
 handler::getCommandGraph() const {
   if (impl->MGraph) {
     return impl->MGraph;
+  } else if (this->MQueue) {
+    return MQueue->getCommandGraph();
```

Comment:
```suggestion
  }
  if (this->MQueue) {
    return MQueue->getCommandGraph();
```

Reply:
Any particular reason for this change? Just wondering if I'm overlooking any coding standard/style.

Reply:
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return